src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -651,7 +651,6 @@ private static Object getSubSectionName(int subSectionPos) {
 
     private static void validateDescriptionSection(String fileName, String sectionName,
             Node subSection) {
-        // Till https://github.com/checkstyle/checkstyle/issues/5777
         if ("config_filters.xml".equals(fileName) && "SuppressionXpathFilter".equals(sectionName)) {
```

Comment:
we should remove comment together with code or change a reason after "till" or "until".
Please apply to all, if you have problems .... lets discuss.

Reply:
It is ok for this issue. List with incompatible checks only have checks that are truly incompatible, all others were fixed in #5777 
So comments are outdated

Reply:
To be more specific, #5777 had list of checks that are incompatible with Xpath filter and all of them were fixed. Now list of incompatible checks consists of checks that are really incompatible and bunch of javadoc checks, which will be fixed in #5770